circe/src/main/scala/org/http4s/circe/CirceInstances.scala

```diff
@@ -19,7 +19,7 @@ package circe
 
 import cats.data.NonEmptyList
 import cats.effect.Concurrent
-import cats.syntax.either._
+import cats.implicits._
```

Comment:
This notation was superseded by
```suggestion
import cats.syntax.all._
```
in [Cats 2.2.0](https://github.com/typelevel/cats/releases/tag/v2.2.0).

It may still work for older syntax though, but it may not support some newer syntax introduced after 2.2.0.
Therefore, this old way to import the syntax is mostly discouraged.

However, `import cats.syntax.either._` is still pretty legit to use...